component/component/provider.jsonnet

```diff
@@ -383,4 +393,47 @@ local runtimeConfigRef(name) = {
       role,
       rolebinding,
     ],
+  [if params.providers.sql.enabled || params.services.vshn.postgres.enabled then '10_provider_sql']:
```

Comment:
I would not use `params.providers.sql.enabled` all together. Just enabled provider sql for any service that needs it.

Reply:
Fine by me